tools/build-examples.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Taken and adapted from: https://github.com/ajdm/travis-build-children
+
+# Get last child project build number
+BUILD_NUM=$(curl -s 'https://api.travis-ci.org/repos/scalameter/scalameter-examples/builds' | grep -o '^\[{"id":[0-9]*,' | grep -o '[0-9]' | tr -d '\n')
+# Restart last child project build
+curl -X POST https://api.travis-ci.org/builds/$BUILD_NUM/restart --header "Authorization: token "$AUTH_TOKEN
```

Comment:
I think it should also work if you put the variable inside the quotes. But should be fine as-is.